installer/x86_64/install.sh

```diff
@@ -385,6 +385,7 @@ if [ "$install_env" != "sonic" ]; then
     }
 else
     demo_mnt="/host"
+    TAR_EXTRA_OPTION="--numeric-owner"
```

Comment:
The ONIE versions installed on our dev devices are using BusyBox tar v1.20.2 which supports the option, however it is not listed in the help menu. See https://github.com/mozilla-b2g/busybox/blob/master/archival/tar.c#L796